src/apps/Map/components/SearchBar.js

```diff
@@ -53,7 +65,7 @@ const closeSize = 20;
 const SearchReset = styled.div`
   position: absolute;
   right: 10px;
-  top: 12px;
+  top: 15px;
```

Comment:
This block is not ideal IMO. If you want to, you can have a look at making it nicer :)

a. I try to no use absolute and absolute values whenever possible. The top-px don't explain where there come from by themself. That would be better with `top: calc( ( 48px [height of parent] / 2 ) - 10px [or even 20px/2, height of self] );`

b. Having text as X make positioning them just right pretty hard. An SVG would make it a lot easier.

Here is what works better IMO:

```
// On the SearchBar__SearchBarInnerWrapper
    display: flex;
    align-items: center;

// On the SearchBar__SearchBarRest

    position: relative;
    right: 30px;
    border-radius: 100%;
    width: 20px;
    height: 20px;
    background: rgb(221, 221, 221);
    color: rgb(85, 85, 85);
    -webkit-box-pack: center;
    justify-content: center;
    -webkit-box-align: center;
    align-items: center;
    line-height: 100%;
    font-weight: 700;
    font-size: 20px;
    cursor: pointer;
    box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 1px 2px;
    display: inline-flex;
    padding-top: 2px; // those paddings are needed to offset the typo-x which is not optimized to 
    padding-left: 1px;
```

Reply:
I'm with you. I would always avoid hard coding pixels. This was just a quick fix, since it was not centered. Didn't want to change the whole component. 

So I was aware of this, but that's not scope of this PR :)